spring-web-modules/spring-mvc-basics-5/pom.xml

```diff
@@ -73,6 +84,7 @@
         <commons-io.version>1.3.2</commons-io.version>
         <json-path.version>2.7.0</json-path.version>
         <jaxb-runtime.version>2.3.5</jaxb-runtime.version>
```

Comment:
Please refactor the version property using "jakarta.servlet.jsp.jstl" identifier, not "jstl"